book/src/output/tracing.md

```diff
@@ -0,0 +1,113 @@
+`tracing` integration
```

Comment:
We also should mention this chapter in `terminal.md` in `Manual printing` section.